pkg/broker/broker_test.go

```diff
@@ -57,36 +58,64 @@ var _ = Describe("Broker", func() {
 	})
 
 	Context("Services", func() {
+		var expectedService brokerapi.Service
+
 		BeforeEach(func() {
-			cfg, err = config.NewConfig("../config/assets/service_config.json")
+			cfg, err = config.NewConfig(
+				"../config/assets/test_config.json",
+				config.ServiceFlags{
+					"../config/assets/service_config.json",
+				},
+			)
 			Expect(err).NotTo(HaveOccurred())
+
+			True := true
+			expectedService = brokerapi.Service{
+				ID:          "some-id",
+				Name:        "eth",
+				Description: "some-desc",
+				Bindable:    true,
+				Tags:        []string{"eth", "geth"},
+				Metadata: &brokerapi.ServiceMetadata{
+					DisplayName:         "some-name",
+					LongDescription:     "some-long-desc",
+					ProviderDisplayName: "some-provider-display-name",
+				},
+				DashboardClient: &brokerapi.ServiceDashboardClient{
+					ID:     "some-client-id",
+					Secret: "some-secret",
+				},
+				Plans: []brokerapi.ServicePlan{
+					brokerapi.ServicePlan{
+						ID:          "some-plan-id",
+						Name:        "free",
+						Description: "free-trial",
+						Free:        &True,
+						Metadata: &brokerapi.ServicePlanMetadata{
+							DisplayName: "service-plan-metadata",
+							Costs: []brokerapi.ServicePlanCost{
+								brokerapi.ServicePlanCost{
+									Amount: map[string]float64{"usd": 1.0},
+									Unit:   "monthly",
+								},
+							},
+							Bullets: []string{"dedicated-node", "another-node"},
+							AdditionalMetadata: map[string]interface{}{
+								"container": map[string]interface{}{
+									"backend": "docker",
+									"image":   "some-image",
+								},
+							},
+						},
+					},
+				},
+			}
 		})
 
 		It("should return service definition", func() {
 			services, err := blockhead.Services(ctx)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(services).To(HaveLen(1))
```

Comment:
Is it possible to not add this in the previous commit and have the removal in this commit?

Reply:
yeah. honestly I am disliking this idea of having multiple commits under one PR. makes it super confusing and hard to review.